preparing plan to add new couchdb node in az-b production

prem-fissionhq opened the pull request:
<!--- Provide a link to the ticket or document which prompted this change -->
https://dimagi-dev.atlassian.net/browse/SAAS-11856
##### ENVIRONMENTS AFFECTED
<!--- list which environments are affected by this change or None if this doesn't change any environment files -->

Plan to add a new CouchDB node in az-b and we will be using snapshot-based copying of data onto this new node before running the migration process which is a manual process.

Thanks,

dannyroberts commented on environments/production/terraform.yml:
```diff
@@ -110,6 +110,17 @@ servers:
     group: "couchdb2_proxy"
     os: ubuntu_pro_bionic
 
+  - server_name: "couch11-production"
```
Not a big deal, but curious why 11 and not 15 which would be the logical next number to pick.

prem-fissionhq replied:
@dannyroberts we just wanted it to be in serial. Instead of 12,13,15,16... if we want to have 11,12,13...

dannyroberts commented on environments/production/terraform.yml:
```diff
@@ -110,6 +110,17 @@ servers:
+    network_tier: "db-private"
+    az: "b"
+    volume_size: 60
+    volume_encrypted: no
```
should be `volume_encrypted: yes` here too.

prem-fissionhq replied:
Danny, I will discuss this with you once.

prem-fissionhq replied:
@dannyroberts I'm sorry missed this notification. 
yes, the root volume also has to be encrypted but I didn't notice the volume_encrypted parameter is in two places.
Yesterday, I realized it and looked into ways to encrypt the root volume. Seems like the best possible solution is to - take a snapshot 
- create a new volume from the snapshot 
- stop the instance and attach the new volume as the root volume to the instance.
- Finally, import the changes to the state file.

dannyroberts replied:
OK it's not urgent, so why don't you just make a ticket for this and we can do it after all other couch work is done

prem-fissionhq replied:
Sure, Danny. I will create a ticket for this task.

dannyroberts reviewed (commented):
I approve pending the `volume_encrypted` change

prem-fissionhq commented:
We have removed couch14 from the Proxy and placed it under maintenance. And added a new node couch11 into the cluster, updated proxy to accept active traffic.
